aioamqp/__init__.py

```diff
@@ -47,6 +49,14 @@ def connect(host='localhost', port=None, login='guest', password='guest',
         if not verify_ssl:
             ssl_context.check_hostname = False
             ssl_context.verify_mode = ssl_module.CERT_NONE
+        for file in ssl_options:
+            if os.path.exists(ssl_options[file]):
+                raise FileNotFoundError(file)
+        ssl_context.load_cert_chain(
+            ssl_options.get('certfile'),
```

Comment:
According to [ssl.load_cert_chain documentation](https://docs.python.org/3/library/ssl.html#ssl.SSLContext.load_cert_chain), it would probably be best use `ssl_options['certfile']` as it would raise a `KeyError` in case you forgot to pass it to the function rather than passing `None` to `ssl.load_cert_file`